internal/command/command.go

```diff
@@ -146,34 +149,35 @@ func createGateway(host string) (gateway.Gateway, error) {
 // 2. if conf is initialized return host by network flag
 // 3. if conf is not initialized and network flag is provided resolve to coded value for that network
 // 4. default to emulator network
-func resolveHost(state *flowkit.State, hostFlag string, networkFlag string) (string, error) {
+func resolveHost(state *flowkit.State, hostFlag, hostNetworkKeyFlag, networkFlag string) (string, string, error) {
```

Comment:
Is the `hostNetworkKeyFlag` even needed here? judging by method impl it doesn't do anything but return it, so maybe consider removing that.

Reply:
I added validation of the --network-key flag here

https://github.com/onflow/flow-cli/pull/449/commits/51177595cb61d92a32420cd746544b6dfe485cc6